controllers/topics.controller.js

```diff
@@ -1,9 +1,15 @@
 const { selectAllTopics } = require("../models/topics.model");
+const { writeAllDescriptions } = require("../models/docs.model");
 
-function getAllTopics(req, res) {
-    return selectAllTopics().then((topics) => {
-        res.status(200).send({ topics });
-    });
-}
+exports.getAllTopics = (req, res, next) => {
+    let { method } = req;
+    let { path } = req.route;
 
```

Comment:
We don’t need these 2 lines of code. Again, I think you’re overcomplicating things. This getAllTopics controller should simply invoke the selectAllTopics model (which you’re doing below) and then sent the response. You shouldn’t be doing anything with the descriptions within this controller.